influxdb/20-statefulset.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: v1
+items:
+- apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    creationTimestamp: "2020-09-16T15:14:52Z"
```

Comment:
That statefulset yaml comes from 

`kubectl get statefulset` instead of `kubectl get statefulset influx` 

that causes the "items" section to be an array of objects (if there are more than one statefulsets, you will have the definition of all of them). Which can't be used to apply the yaml clean. Please, get the yaml without the list of objects and remove the autogenerated code

- creationTimestmap
- generation
- resourceVersion
- SelfLink
- uid

Reply:
done